src/common/Types.ts

```diff
@@ -590,6 +591,7 @@ export interface ViewPropsShared extends CommonProps, CommonAccessibilityProps {
     importantForLayout?: boolean; // Web-only, additional invisible DOM elements will be added to track the size changes faster
     id?: string; // Web-only. Needed for accessibility.
     ariaLabelledBy?: string; // Web-only. Needed for accessibility.
+    ariaRoleDescription?: string; // Web-only. Needed for accessibility.
```

Comment:
Since you're adding a new prop, this needs to be added to the documentation (view.md). 

It also needs to be added to a test case (e.g. ViewBasicTest.tsx in RXPTest).

Reply:
Done.